syncserver/__init__.py

```diff
@@ -31,13 +35,16 @@ def includeme(config):
     """Install SyncServer application into the given Pyramid configurator."""
     # Set the umask so that files are created with secure permissions.
     # Necessary for e.g. created-on-demand sqlite database files.
-    os.umask(0077)
+    os.umask(0o077)
 
     # If PyOpenSSL is available, configure requests to use it.
     # This helps improve security on older python versions.
     if HAS_PYOPENSSL:
         requests.packages.urllib3.contrib.pyopenssl.inject_into_urllib3()
 
+    def generate_random_hex_key(length):
+        return binascii.hexlify(os.urandom(length // 2))
```

Comment:
Please make this a top-level function rather than defining it inline in `includeme`.